manifests/plugin/remote_execution/ssh.pp

```diff
@@ -49,6 +52,18 @@
     template_path => 'foreman_proxy/plugin/remote_execution_ssh.yml.erb',
   }
 
+  if $ssh_kerberos_auth {
+    if $::osfamily == 'RedHat' and $::operatingsystem != 'Fedora' {
+      $scl_prefix = 'tfm-'
+    } else {
+      $scl_prefix = '' # lint:ignore:empty_string_assignment
+    }
+
+    foreman_proxy::plugin { 'net-ssh-krb':
```

Comment:
Good point about `package`, will do. I'll have to look into the debian naming, debian packaging pr is [here](https://github.com/theforeman/foreman-packaging/pull/1664)